src/serializer/serializer.js

```diff
@@ -1245,6 +1323,7 @@ export class Serializer {
         for (let instance of instances) {
           let { functionValue, serializedBindings } = instance;
           let id = this._getValIdForReference(functionValue);
+          invariant(id.type === "Identifier");
```

Comment:
This code change allows for _getValIdForReference to return a BabelNodeMemberExpression. I'll add a comment why it's an identifier in this case.